CauseContract.sol

```diff
@@ -50,29 +86,55 @@ contract CauseContract {
     }
 
     function retrieveInfo() public view returns (ContractInfo memory) {
-        return ContractInfo(id, admin, incoming, outgoing, contractAddress);
+        return ContractInfo(id, admin, incoming, outgoing, contractAddress, causeTotal, endCause, emailAddress, causeDescription, websiteURL, thumbnail);
```

Comment:
Could you change this to thumbnailURL just to make it clear what we should be expecting from it?